DOCS/man/options.rst

```diff
@@ -20,19 +20,18 @@ Track Selection
           audio.
 
 ``--slang=<languagecode[,languagecode,...]>``
-    Specify a priority list of subtitle languages to use. Different container
-    formats employ different language codes. DVDs use ISO 639-1 two letter
-    language codes, Matroska uses ISO 639-2 three letter language codes while
-    OGM uses a free-form identifier. See also ``--sid``.
+    Equivalent to ``--alang``, for subtitle tracks.
```

Comment:
what about the doc changes for that?
since we match different tags against eachother it is no longer relevant for the user to know which container uses what, he can simply use either `pol` or `pl`

Reply:
changes